### ISSUE TITLE
Tests fail using Modular<Integer> only in DEBUG mode

### ISSUE THREAD

**Issue by ClementPernet** [linbox-team/fflas-ffpack#67]
`test-lu`, `test-rankprofiles`, `test-echelon` fail consistently when --enable-debug is passed to configure, but don't fail without the debug option.
log files show failures of the fgem_checkers or trsm_checkers. For example `test-echelon.log` ends with:
```
...
Checking ...ModularBalanced<int64_t> mod 26093831 ... PASSED 
Checking ........ModularBalanced<int64_t> mod 479 ... PASSED 
terminate called after throwing an instance of 'FailureFgemmCheck'
Checking Modular<Integer> modulo 234142783307639011054589950880506388407 ... FAIL test-rankprofiles (exit status: 134)
```

**Comment by ClementPernet** [comment]
related to #29 according to @pascalgiorgi

**Comment by pascalgiorgi** [comment]
This has been fixed with merge #90
